### PR TITLE
DDF-2892 added template for appendix to docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Adds the appendices to the templated documentation. (jdocumentation)

#### Who is reviewing it? 

@mcalcote @vinamartin @emmberk @Lambeaux @jrnorth 

#### Choose 2 committers to review/merge the PR. 

@clockard
@coyotesqrl
@lessarderic
@pklinef
@ricklarsen - Documentation
@shaundmorris

#### How should this be tested? 

- documentation builds successfully
- all appendices are included in jdocumentation.html/.pdf

#### Any background context you want to provide?

"New" adoc files are existing content with headers added. Content review is *Optional*

#### What are the relevant tickets?
[DDF-2892](https://codice.atlassian.net/browse/DDF-2892)

#### Screenshots (if appropriate)

[jdocumentation.pdf](https://github.com/codice/ddf/files/1231807/jdocumentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/jdocs/content/_metadata-attributes/core-attributes-table-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,157 @@
+:title: Core Attributes
+:type: subappendix
+:parent: Catalog Taxonomy
+:status: published
+:summary: Core Attributes
```
Comment (emmberk) [inline distribution/docs/src/main/jdocs/content/_metadata-attributes/core-attributes-table-contents.adoc]: Missing period

File: distribution/docs/src/main/jdocs/content/dependency-list/ddf-dependency-list-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,285 @@
+:title: Dependency List
+:type: appendix
+:status: published
+:parent: ${ddf-branding} Dependency List
+:children: none
+:order: 00
+:summary: ${ddf-branding} Dependency List
```
Comment (emmberk) [inline distribution/docs/src/main/jdocs/content/dependency-list/ddf-dependency-list-contents.adoc]: Missing period.

File: distribution/docs/src/main/jdocs/content/_metadata-attributes/core-attributes-table-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_metadata-attributes/core-attributes-table-contents.adoc]: fixed.

File: distribution/docs/src/main/jdocs/content/dependency-list/ddf-dependency-list-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/dependency-list/ddf-dependency-list-contents.adoc]: fixed.

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
@emmberk The template version will be replacing the manually defined version some time in the next few weeks, so I only applied this change to the template version. It won't show up in the admin console until https://codice.atlassian.net/browse/DDF-3240 is completed.

**Pull request merged**